backend/src/main/java/com/benine/backend/http/EditPresetHandler.java

```diff
@@ -0,0 +1,105 @@
+package com.benine.backend.http;
+
+import com.benine.backend.Preset;
+import com.benine.backend.ServerController;
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.ipcameracontrol.IPCamera;
+import com.benine.backend.video.StreamNotAvailableException;
+
+import org.eclipse.jetty.server.Request;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/**
+ * Class that handles editing presets.
+ */
+public class EditPresetHandler extends RequestHandler {
+  
+  /**
+   * Constructor for a new editPresetHandler that handles editing a preset.
+   */
+  public EditPresetHandler() {}
+    
+  private ServerController control = ServerController.getInstance();
+
+  @Override
+  public void handle(String s, Request request, HttpServletRequest req, HttpServletResponse res)
+          throws IOException, ServletException {
+    
+    try {
+      String overwriteTag = request.getParameter("overwriteTag");
+      String overwritePosition = request.getParameter("overwriteposition");
+      String camID = request.getParameter("camera");
```

Comment:
Why do you request the cameraID from the client? I think the presetID is enough because when you have the preset you can do preset.getCameraId(), this way the client does not have to send redundant information.